src/dataflow/render/mod.rs

```diff
@@ -602,8 +602,15 @@ pub(crate) fn build_dataflow<A: Allocate>(
                         sink.from.0,
                         sink.from.1.typ().clone(),
                     ))
+                    // What does this even mean?
```

Comment:
the `collection` function looks like it looks in dataflow local collections / arrangements and shared arrangements to find a matching thing for this RelationExpr and returns None if it can't find anything. Might be good to edit the message to something like "No arrangement or collection found to service relation for sink {}"...